styles.css

```diff
@@ -35,6 +39,22 @@ p {
   justify-content: space-between;
 }
 
+/* for hamburger */
+
+.mobile-link {
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  line-height: 44px;
+  font-size: 32px;
+  color: #3a4a42;
+  transition: 0.7s ease;
+  text-decoration: none;
+}
+
+.mobile-link:hover {
+  color: #36b37f;
+}
+
 /* for logo(welcome) styling */
 
 .logo {
```

Comment:
- [ ] You did a great job on the project functionality but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=XiHXj9otHoCPVw2x-0) design as much as possible. Please fix the below issues and make sure your design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=XiHXj9otHoCPVw2x-0) design.
  - Please decrease the left padding for the mobile menu options.
  - Please decrease the gap between the mobile menu options.
  - Please use the correct color for the close(X) button `#A7B8B0`.

### Your design
![image](https://user-images.githubusercontent.com/83803414/223148661-59488187-1ab5-4db1-8799-ca0b70dace09.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/223148751-70917a99-3ee4-47c0-a57e-3d65ff545071.png)